tests/dummy/config/targets.js

```diff
@@ -1,3 +1,27 @@
+'use strict';
+
+const browsers = [
```

Comment:
I would suggest using the same targets we use in our production applications:

```
let browsers = [
  'last 1 Chrome versions',
  'last 1 Firefox versions',
  'last 1 Safari versions'
];

const isCI = Boolean(process.env.CI);
const isProduction = process.env.EMBER_ENV === 'production';

if (isCI || isProduction) {
  browsers = [
    'last 3 Chrome versions',
    'last 3 iOS versions',
    'last 2 ChromeAndroid versions',
    'last 3 Firefox versions',
    'last 2 Safari versions',
    'last 3 Edge versions',
    'IE 11'
  ]
}
```

Reply:
This applies to all CI so it fails with the newer scenarios (ember 4.x), which don't support IE11.

Reply:
Ah right, I missed the fact that this was just for tests/dummy -- the consuming applications will compile as necessary according to their own configs.